src/main/java/org/dimdev/riftloader/launch/RiftLoaderTweaker.java

```diff
@@ -32,7 +32,7 @@ public void injectIntoClassLoader(LaunchClassLoader classLoader) {
         // classes may not be equal, and 'instanceof' may return false when it should be true.
         try {
             Class<?> clazz = Launch.classLoader.findClass("org.dimdev.riftloader.RiftLoader");
-            clazz.getMethod("load").invoke(clazz.getField("instance").get(null));
+            clazz.getDeclaredMethod("load").invoke(clazz.getField("instance").get(null));
```

Comment:
`getMethod`, not `getDeclaredMethod`, it's public.

Reply:
`getDeclaredMethod` will only accept methods from that class (public AND private), while `getMethod` will accept ALL (public) methods, including superclass methods.

Reply:
Yes, and the second is what we want. `getMethod` is the standard way to get a public method in Java.
`getDeclaredMethod` is used to bypass visibility checks and get all methods (if the security manager allows the `accessDeclaredMembers` permission).